php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1403,15 +1405,16 @@ abstract class NDB_BVL_Instrument extends NDB_Page
     {
         $allValues  = $this->getInstanceData();
         $dictionary = $this->getDataDictionary();
-
+        if (!empty($dictionary)){
```

Comment:
This shouldn't be changed.  If the data dictionary is empty, the field doesn't exist, and the exception is the appropriate behaviour.